trace/trace.go

```diff
@@ -172,6 +201,42 @@ func (t *Trace) Send() {
 	}
 }
 
+// SpanContext contains information related to a Span. Context should be carried
+// around by the currently active span, and optionally propagated in trace context
+// headers.
+type SpanContext struct {
+	TraceID      string
+	ParentID     string
+	TraceContext map[string]interface{}
+	Dataset      string
+	TraceState   byte
```

Comment:
Need a comment on TraceState describing what it is. Maybe the others too? But this one for sure.